test/functional/tests/lazy_writes/test_lazy_writes_signals.py

```diff
@@ -0,0 +1,240 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+from aenum import Enum
+from time import sleep
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode, CacheModeTrait, FlushParametersAlru, Time
+from scsi_debug_fua_signals import ScsiDebugFua
+from storage_devices.disk import DiskType, DiskTypeSet
+from core.test_run import TestRun
+from test_tools.disk_utils import Filesystem
+from test_tools.fs_utils import create_random_test_file
+from test_utils import os_utils
+from test_utils.size import Size, Unit
+from tools.custom_kernel_module import load_custom_kernel_module
+
+mount_point = "/mnt/cas"
+log_msg = "/var/log/messages"
+
+
+@pytest.mark.require_plugin("scsi_debug_fua_signals")
+@pytest.mark.parametrize("cache_mode", CacheMode.with_traits(CacheModeTrait.LazyWrites))
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+def test_flush_signal_core(cache_mode):
+    """
+        title: Test for FLUSH signals to core device in modes with lazy writes.
```

Comment:
Why only FLUSH and not FUA?

Reply:
This test does not require checking for FUA signals.